harness/libraries/layout_of_apps_directory.py

```diff
@@ -209,15 +209,15 @@ def get_path_to_workspace(self):
     # Returns the path to the test workspace build directory.
     #
     def get_path_to_workspace_build(self):
```

Comment:
This seems fine either way, I don't think an empty path is possible here, but can't hurt.